ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -155,7 +157,11 @@ const primaryButtonPropsFromState = (
     }),
     c => ({
       ...baseButtonProps,
-      disabled: !isCardNumberValid || !isCvvValid,
+      disabled:
+        !isCardNumberValid ||
+        !isCvvValid ||
+        !isHolderValid ||
+        !isExpirationDateValid,
```

Comment:
```suggestion
      disabled:[
        isCardNumberValid,
        isCvvValid,
        isHolderValid,
        isExpirationDateValid
      ].some(v => !v),
```
it's pretty the some but from my POV, it easier to read